.copr/Makefile

```diff
@@ -0,0 +1,9 @@
+srpm:
+	dnf -y install git
+	git submodule init
+	git submodule update
```

Comment:
Should be enough to do
```suggestion
	git submodule update --init --recursive
```
instead of `submodule init` + `submodule update`, plus a `--recursive` is needed to be on the safe side

Reply:
Are you saying that the `git submodule init` isn't needed if I add `--init` as you suggest above?

Reply:
Yes, `git submodule update --init` does that for you.

Reply:
Fixed